tests/validate/bandwidth_test/src/host.cpp

```diff
@@ -77,6 +78,17 @@ int main(int argc, char** argv) {
 
         temp = loadPtreeRoot.get_child("total_ddr_banks");
         NUM_KERNEL = temp.get_value<int>();
+        NUM_KERNEL_DDR = NUM_KERNEL;
+        boost::property_tree::ptree ptMemArray;
+        ptMemArray = loadPtreeRoot.get_child("meminfo");
+        for (auto memEntry : ptMemArray) {
+            boost::property_tree::ptree ptMemEntry = memEntry.second;
+            const std::string& sValue = ptMemEntry.get<std::string>("type");
```

Comment:
`auto sValue = ...;`   You are not getting a reference to any internally held data, so a copy is made any way.